packages/@n8n/db/package.json

```diff
@@ -26,13 +26,16 @@
     "@n8n/permissions": "workspace:^",
     "@n8n/typeorm": "catalog:",
     "class-validator": "0.14.0",
+    "flatted": "catalog:",
+    "lodash": "catalog:",
     "n8n-core": "workspace:^",
     "n8n-workflow": "workspace:^",
     "nanoid": "catalog:",
     "reflect-metadata": "catalog:",
     "xss": "catalog:"
   },
   "devDependencies": {
-    "@n8n/typescript-config": "workspace:*"
+    "@n8n/typescript-config": "workspace:*",
+    "jest-mock-extended": "^3.0.4"
```

Comment:
this isn't technically needed. All packages inherit this as a dev dependency from the root package.json.